sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/DataFeedSources/SqlServerDataFeedSource.cs

```diff
@@ -17,14 +17,37 @@ public class SqlServerDataFeedSource : DataFeedSource
         private string _connectionString;
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="SqlServerDataFeedSource"/> class.
+        /// Initializes a new instance of the <see cref="SqlServerDataFeedSource"/> class. This constructor does not
+        /// set a <see cref="ConnectionString"/>, so you must assign an <see cref="AuthenticationType"/> to the
+        /// <see cref="Authentication"/> property. Currently, only the <see cref="AuthenticationType.SqlConnectionString"/>
+        /// authentication is supported without a connection string. If you intend to use another type of authentication,
+        /// see <see cref="SqlServerDataFeedSource(string, string)"/>.
+        /// </summary>
+        /// <param name="query">The query to retrieve the data to be ingested.</param>
+        /// <exception cref="ArgumentNullException"><paramref name="query"/> is null.</exception>
+        /// <exception cref="ArgumentException"><paramref name="query"/> is empty.</exception>
+        public SqlServerDataFeedSource(string query)
```

Comment:
do u want to add a test for it? I think as is fine, but just in case u want to check query empty and the exception and etc...

Reply:
Added tests for new ctors.